src/tallies/tally.cpp

```diff
@@ -250,6 +254,42 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), 1001) !=
+                std::end(scores_);
+  if (exists == 1) {
+    std::cout << "PULSE HEIGHT TALLY SIMULATION, VERSION 1.0, 15.09.2021"
+              << std::endl;
+  }
+  if (exists == 1 && scores_.size() > 1) {
+    std::cout << "Error: The Pulse-Height Tally can currently not be used in "
+                 "combination with other tallys."
+              << std::endl;
+    exit(0);
```

Comment:
1) If it's an error, you maybe shouldn't return 0 as an exit code since that generally means stuff is all good.
2) Probably should do `openmc::fatal_error` here rather than using cout then exit(0).

Reply:
yes, I changed that.